source/change-streams/tests/unified/change-streams-disambiguatedPaths.yml

```diff
@@ -0,0 +1,102 @@
+description: "disambiguatedPaths"
+schemaVersion: "1.9"
```

Comment:
Why do you need schema version 1.9? I think we can use a lower minor version unless there's some feature of this test I'm overlooking. These might be able to use schema version 1.0, but let me know what you think.

Reply:
Yeah, good point.  Since these tests pass on load-balanced topologies (see my other comment), I lowered the schema version to 1.3.